test/test_jobrunner.py

```diff
@@ -357,3 +357,30 @@ def test_run_slurm(self, mock_njs, mock_auth):
         jr._get_token_lifetime = MagicMock(return_value=self.future)
         out = jr.run()
         self.assertNotIn('error', out)
+
+    @attr('offline')
+    @patch('JobRunner.JobRunner.KBaseAuth', autospec=True)
+    @patch('JobRunner.JobRunner.NJS', autospec=True)
+    def test_run_wdl(self, mock_njs, mock_auth):
+        self._cleanup(self.jobid)
+        params = deepcopy(NJS_JOB_PARAMS)
+        params[0]['method'] = 'RunTester.run_RunTester'
+        params[0]['params'] = [{'do_wdl': 1}]
+        params[1]['auth-service-url'] = self.config['auth-service-url']
+        params[1]['auth.service.url.v2'] = self.config['auth2-url']
+        jr = JobRunner(self.config, self.njs_url, self.jobid, self.token,
+                       self.admin_token)
+        mlog = MockLogger()
+        jr.logger = mlog
+        jr.sr.logger = mlog
+        rv = deepcopy(CATALOG_GET_MODULE_VERSION)
+        rv['docker_img_name'] = 'test/runtester:latest'
+        jr.cc.catalog.get_module_version = MagicMock(return_value=rv)
+        jr.cc.catalog.list_volume_mounts = MagicMock(return_value=[])
+        jr.cc.catalog.get_secure_config_params = MagicMock(return_value=None)
+        jr.njs.get_job_params.return_value = params
```

Comment:
need to set    jr.njs.get_job_params.return_value = params[0]
need to set list_config() = params[1]